pypeit/wavetilts.py

```diff
@@ -417,71 +437,35 @@ def load(self, ifile=None, return_header=False):
         Args:
             ifile (:obj:`str`, optional):
                 Name of the master frame file.  Defaults to
-                :attr:`file_path`.
+                :attr:`master_file_path`.
             return_header (:obj:`bool`, optional):
                 Return the header.
 
         Returns:
-            Returns the tilts dictionary.  If return_header is
-            true, the primary header is also returned.  If nothing is
+            dict: Returns the tilts dictionary.  If nothing is
             loaded, either because :attr:`reuse_masters` is `False` or
             the file does not exist, everything is returned as None (one
             per expected return object).
         """
-        # Format the input and set the tuple for an empty return
-        _ifile = self.file_path if ifile is None else ifile
-        empty_return = (None, None) if return_header else None
-
-        if not self.reuse_masters:
-            # User does not want to load masters
-            msgs.warn('PypeIt will not reuse masters!')
-            return empty_return
-
-        if not os.path.isfile(_ifile):
-            # Master file doesn't exist
-            msgs.warn('No Master {0} frame found: {1}'.format(self.master_type, self.file_path))
-            return empty_return
-
-        # Read and return
-        msgs.info('Loading Master {0} frame: {1}'.format(self.master_type, _ifile))
-        return self.load_from_file(_ifile, return_header=return_header)
-
-    @staticmethod
```

Comment:
See my comment above. I don't want to load in the WaveTilts class from file as you have now changed things. I just want the tilts_dict. The load_from_file method below is a simple one line interface to read in the master from the file. The WaveTilts object is not used in PypeIt as a master, but the tilts_dict is. I don't need to instantiate a class, I want to read in that dict. Please add load_from_file functionality back to this class.

Reply:
See below